### PR TITLE
Fixes #12757: update packages to use host instead of system.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/12757

### CODE REVIEW

File: lib/hammer_cli_katello/output/fields.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (cfouant) [inline lib/hammer_cli_katello/output/fields.rb]: Is there a reason for this?

**Comment by cfouant** [comment]
ACK from me pending the requested nitpick change

**Comment by jlsherrill** [comment]
@waldenraines ping, plz2merge

**Comment by jlsherrill** [comment]
merging, thanks @waldenraines !

**Pull request merged**